### ISSUE TITLE
On IBM credential

### ISSUE THREAD

**Issue by tbcdebug** [Qiskit/qiskit-textbook#14]
This is common for chapter 3 (algorithm) and 4 (application). Currently, real quantum computers are accessed in this way. 
```
IBMQ.load_accounts()
backend = IBMQ.get_backend('ibmq_16_melbourne')
```

I believe this is a somehow older way (?) and do not work anymore, probably should change to the following?

```
IBMQ.load_account()
provider = IBMQ.get_provider(hub='ibm-q')
backend = provider.get_backend('ibmq_16_melbourne')
```

**Comment by aasfaw** [comment]
This is correct @tbcdebug. Would you like to open a pull request with these changes?

**Comment by tbcdebug** [comment]
Yes

**Comment by tbcdebug** [comment]
I've already created a pull request.

**Comment by aasfaw** [comment]
Great job @tbcdebug -- I've left you some comments there. Just a couple of things and it's ready to merge.

**Comment by aasfaw** [comment]
Closed by #31
